controllers/userControllers.js

```diff
@@ -34,9 +34,9 @@ exports.createUser = async (req, res) => {
 
     res.status(201).json(newUser);
     
-  } catch (err) {
+  } catch (error) {
     console.error("Error creating user:", error);
-    res.status(500).json({ message: "Internal server error" });
+    res.status(500).json({ message: "Goiania is the best"});
```

Comment:
1. This is a controller. If you keep this code in here, this would be the message being sent to users when there are errors with the database.
2. I though you were going to use `error.message` instead of hardcoding a message. If you are hard coding, please be explicit with something like, "Error creating user. Please try again".